conf/application.conf

```diff
@@ -183,4 +183,4 @@ accessibility-statement {
    service-path = "/guaranteed-minimum-pension"
 }
 
-isMongoDBCacheEnabled = false
\ No newline at end of file
+isMongoDBCacheEnabled = true
```

Comment:
Change back to false, but add a false to app-config-production first and then can change it